DOCS/request-new-license.md

```diff
@@ -3,28 +3,29 @@ The SPDX Legal Team appreciates proposals for new free and open source licenses
 
 # How to request a new license or exception be added to the SPDX License List
 
-1. Join the [legal team mailing list](https://lists.spdx.org/g/spdx-legal).
+1.  Review the [license inclusion principles](license-inclusion-principles.md). Please refrain from submitting licenses that clearly do not meet these principles, for example, a license for which the text is not finalized or is only used on your personal project.
 
-2.  Review the [license inclusion principles](license-inclusion-principles.md).
+    1.  Check the [SPDX License List](https://spdx.org/licenses/), keeping in mind the [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) and issues labeled [new license/exception request](https://github.com/spdx/license-list-XML/labels/new%20license%2Fexception%20request) to ensure your license or exception has not been previously requested.
```

Comment:
URL is 404 for the matching guidelines. Current link from the SPDX License List page to the matching guidelines is https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/

Reply:
fixed